poi-ooxml/src/main/java/org/apache/poi/poifs/crypt/temp/EncryptedTempData.java

```diff
@@ -46,8 +46,9 @@
  */
 @Beta
 public class EncryptedTempData {
-    private static Logger LOG = LogManager.getLogger(EncryptedTempData.class);
+    public static final SecureRandom RANDOM = new SecureRandom();
```

Comment:
can you move this somewhere else? shouldn't be public in this `@Beta` internal class